src/components/reusables/Form.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import SaveButton from "./SaveButton";
+
+export interface FormProps {
+  content: Array<JSX.Element>;
+  onSubmit: any;
+  hiddenName?: string;
+  hiddenValue?: string;
```

Comment:
in theory a form could have multiple hidden inputs. i'm not sure this needs to be separated out specifically, it could be part of the content.

Reply:
It will always need to have at least one hidden input, though...right?  I figure any additional hidden inputs can be part of the content...

Reply:
I don't know that it necessarily would, it would depend on the form. Creating a new thing would be less likely to have one than editing an existing thing (that may not apply for the registry though)

Reply:
To my knowledge, the registry interface is never going to have a creating-a-new-thing form.  Worth leaving alone for now, and then reassessing if we ever try to reuse this component elsewhere (maybe give it an optional `edit` prop to indicate whether it should have the hidden input)?

Reply:
That may be right, but I think it's also more of creating generic and reuseable components without depending just on the use case. Then it'll be easier to update and extend later on than to have coupled components that will require more work to extend.